src/components/AsideSection/AsideSection.tsx

```diff
@@ -27,11 +27,11 @@ export default class AsideSection extends React.Component<AsideSectionProps, {}>
   render() {
     return (
       <div className="y-aside-section">
-        <Block textSize={TextSize.SMALL}>
+        <Block>
           <FixedGridRow className="y-aside-section--header" bottomSpacing={GutterSize.SMALL}>
-            <FixedGridColumn>
+            <FixedGridColumn fixed={true}>
```

Comment:
why are we moving the smaller column to no longer be fixed?

Reply:
With a fixed={true} fake link is aligned to right, even though I set text-align: right;
I just played with it for a while, tested different scenarios and came to this combination of CSS and components properties